packages/base/src/utils/constants.ts

```diff
@@ -941,6 +948,23 @@ export interface AggregatorInfo {
   invert?: boolean;
 }
 
+export const CHRONICLE_METH_USD_SCRIBE_MAP: Record<Network.Mantle, string> = {
+  [Network.Mantle]: '0xfabBc20156309A0f0270ceab67EC150e537C5EDF',
+};
+
+export const CHRONICLE_PRICE_SCRIBES_MAP: Record<Network, Record<string, string>> = {
```

Comment:
Let's change this to use an object with the following fields { `scribeAddress`, `tokenPairAddress` (which defaults to `undefined` if USD, else a token pair) }